### PR TITLE
Added Permission Workaround 

### PR DESCRIPTION
**Pull request by Scouttp**
Installation Updated - requires root to install

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ Installation process
    As an alternative you can get the source code from GitHub_ and install it
    using the setup script::
 
-    python setup.py install
+    sudo python setup.py install
```
Comment (Scouttp) [inline docs/installation.rst]: The install command requires root permissions

File: update.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+git merge upstream/master
+git pull
+git commit
+git push
```
Comment (linusg) [inline update.sh]: Thanks for the PR! Just one question: what do we need `update.sh` for? It's basically for updating the local repository, right?

Comment (Scouttp) [inline update.sh]: You dont have to worry about this file but for others it pulls from your repository, then theirs and commits it all back to theirs. Mostly it just keeps their fork up to date with yours.

Comment (linusg) [inline update.sh]: @Scouttp - Alright, merged. Thank you!

**Pull request merged**